src/platform/user/profile/vet360/containers/TimedAlertBox.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+class TimedAlertBox extends React.Component {
+  state = {
+    isVisible: this.props.startTimer,
+  };
+
+  componentDidMount() {
+    if (this.props.startTimer) {
+      if (this.intervalId === undefined) {
```

Comment:
maybe move this up into the check on line 10? so `if (this.props.startTimer && !this.intervalId)`? Although checking for `this.intervalId` seems unnecessary in this case because it will always be undefined when the component mounts?